topo/topo.go

```diff
@@ -210,10 +212,13 @@ func (m *Manager) Push(ctx context.Context) error {
 		if err := n.CreateService(ctx); err != nil {
 			return err
 		}
-		if err := n.CreatePod(ctx); err != nil {
```

Comment:
you are breaking all other implementations if you do this
if you are going to suggest this change then also update all implementations in the same PR